code/modules/mob/mob.dm

```diff
@@ -1287,6 +1287,24 @@ var/list/slot_equipment_priority = list( \
 			var/mob/living/carbon/human/H = M
 			H.handle_regular_hud_updates()
 
+/mob/verb/rotate_chair() //To allow for actually sensible chair selection
```

Comment:
Alright, I'll figure out the least shitty way to do that and do it later today probably